src/main/scala/io/sdkman/changelogs/ZnaiMigrations.scala

```diff
@@ -0,0 +1,23 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "049")
+class ZnaiMigrations {
+  @ChangeSet(
+    order = "001",
+    id = "001_add_znai_candidate",
+    author = "MykolaGolubyev"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "znai",
+      name = "znai",
```

Comment:
This is the "pretty" name, so how about we capitalise this as it appears in the copy of your description?

Reply:
we didn't do it for webtau. I can remove capitalization in the text below for consistency.